src/cargo/util/process_builder.rs

```diff
@@ -129,11 +131,11 @@ impl ProcessBuilder {
     }
 }
 
-pub fn process<T: AsRef<OsStr>>(cmd: T) -> CargoResult<ProcessBuilder> {
+pub fn process<T: AsRef<OsStr>, U: AsRef<OsStr>>(cmd: T, cwd: U) -> CargoResult<ProcessBuilder> {
```

Comment:
I'd prefer to stick to a `Command`-like API and from looking around it's actually relatively uncommon to switch to anything other than the already current-directory.